scripts/troubleshooting.sh

```diff
@@ -39,21 +62,49 @@ detect_distro() {
 
 panel_logs(){
   if [ -f "/var/www/pterodactyl/storage/logs/laravel-$(date +%F).log" ]; then 
-    panel_log="$(tail -n 100 /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | nc bin.ptdl.co 99)"
+    panel_log="$(nc termbin.com 9999 < /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | tr -d '\0')"
   else
     panel_log='Empty'
   fi
 }
 
+wings_logs(){
+  if [ -f "/var/log/pterodactyl/wings.log" ]; then
+    wings_log="$(nc termbin.com 9999 < /var/log/pterodactyl/wings.log | tr -d '\0')"
```

Comment:
Linked to my previous comment :slightly_smiling_face:

Reply:
Fixed